templates/al2023/provisioners/install-nodeadm.sh

```diff
@@ -6,6 +6,10 @@ set -o errexit
 
 sudo systemctl start containerd
 
+# generate and store containerd version in file /etc/containerd/containerd-version.txt
+CONTAINERD_VERSION=$(yum list installed | grep containerd | awk '{print $2}')
+echo $CONTAINERD_VERSION | sudo tee /etc/containerd/containerd-version.txt
```

Comment:
if we do this it should be under `/etc/eks` as a more proper location for EKS-aware services

```suggestion
echo $CONTAINERD_VERSION | sudo tee /etc/eks/containerd-version.txt
```